encrypted-media-respec.html

```diff
@@ -2024,9 +2024,21 @@
 
     <section>
       <h2><a>HTMLMediaElement</a> Extensions</h2>
+      
+      <!-- Put all the monkey-patching here -->
+      
       <p>This section specifies additions to and modifications of the <a def-id="htmlmediaelement"></a> [[!HTML5]] when the Encrypted Media Extensions are supported.</p>
+      
+      <p>This specification adds two internal values to the <a def-id="htmlmediaelement"></a>: <var>waiting for key</var> which shall have a boolean value and <var>encrypted block queue</var> which shall be a queue of encrypted blocks awaiting decryption.</p>
 
-      <p>When a <a def-id="htmlmediaelement"></a> is created, its internal <var>waiting for key</var> value is initialized to <code>false</code>.</p>
+      <p>When a <a def-id="htmlmediaelement"></a> is created, its internal <var>waiting for key</var> value shall initialized to <code>false</code> and its <var>encrypted block queue</var> value shall be empty.</p>
+      
+      <p>In addition to the criteria specified in [[!HTML5]] an <a def-id="htmlmediaelement"></a> shall be considered a <a def-id="blocked-media-element"></a> if its <var>waiting for key</var> value is <code>true</code>.</p>
+      
+      <p>When the media element encounters <a def-id="initialization-data"></a> in the <a def-id="media-data"></a> during the <a def-id="resource-fetch-algorithm"></a> the user agent shall run the <a def-id="initdata-encountered-algorithm"></a> algorithm.</p>
```

Comment:
nits:
* Add a comma before "the user agent"
* The media element encounters the initData, but the user agent performs the actions. This seems inconsistent. The former comes from the existing text below, but that may also have been incorrect.